src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/stack-trace.svelte

```diff
@@ -30,37 +24,53 @@
   import EmptyState from '$lib/components/empty-state.svelte';
 
   export let namespace: string;
-  export let stackTrace: string;
   export let workflow: WorkflowExecution;
 
   let currentdate = new Date();
   let isLoading = false;
 
-  const refreshStackTrace = async () => {
-    isLoading = true;
+  const getStackTrace = () =>
+    getWorkflowStackTrace({
+      workflow,
+      namespace,
+    });
+
+  let stackTrace = getStackTrace();
 
-    stackTrace = await getWorkflowStackTrace({
+  const refreshStackTrace = () => {
+    stackTrace = getWorkflowStackTrace({
       workflow,
       namespace,
     });
 
-    isLoading = false;
-    currentdate = new Date();
+    stackTrace.then(() => {
+      currentdate = new Date();
+    });
```

Comment:
This could be await if we update the methods to be async. But I don't think there's a lot of value in that. Just throwing it out there.